packages/usa-combo-box/src/test/combo-box.spec.js

```diff
@@ -220,6 +220,29 @@ tests.forEach(({ name, selector: containerSelector }) => {
         );
       });
 
+      it("should sort matches by options that start with the query, then options that contain the query", () => {
```

Comment:
I originally considered this, but that test is actually to test the `data-filter` property that allows users pass custom regex expressions to the combo-box. I opted for this test since this is the default behavior of the combo box

Reply:
Thanks for clarifying!